packages/pds/src/image/server.ts

```diff
@@ -145,6 +146,8 @@ export class BlobDiskCache implements BlobCache {
         throw new BlobNotFoundError()
       }
       throw err
+    } finally {
+      await handle?.close()
```

Comment:
Naively I would expect this to close the file descriptor in such a way that it prevents consumers from reading the returned stream—do you know why that isn't the case?  Or if that does turn out to be case, maybe this just needs to go in the `catch {}` path?

Reply:
yeah i think you're right i totally missed that 🙃 

Okay i just used `fsSync` to return the readstream